proc_scripts/load_data.py

```diff
@@ -95,43 +243,92 @@ def proc_Hahn_echoph(s, fl=None):
         fl.image(abs(s))
     return s
 
-def proc_spincore_IR(s,fl=None):
+def proc_spincore_IR(s,clock_correction,fl=None):
+    logger.info(strm("pre-processing for IR acquired on spincore"))
     s.rename('vd','indirect')
     s.reorder(['ph1','ph2','indirect','t2'])
     s.ft(['ph2','ph1'])
     s.ft('t2', shift=True)
     if fl is not None:
         fl.next('raw data -- coherence channels')
-        fl.image(s)
+        fl.image(s.C.setaxis('indirect','#').set_units('indirect','scan #'))
     s.ift('t2')
     if fl is not None:
         fl.next('time domain (all $\\Delta p$)')
-        fl.image(s)
+        fl.image(s.C.setaxis('indirect','#').set_units('indirect','scan #'))
     s.ft('t2', pad=4096)
     if fl is not None:
         fl.next('frequency domain (all $\\Delta p$)')
-        fl.image(s)
+        fl.image(s.C.setaxis('indirect','#').set_units('indirect','scan #'))
+    s *= exp(-1j*s.fromaxis('indirect')*clock_correction)
     return s
 
-def proc_nutation(s):
-    logging.info("loading pre-processing for nutation")
+def proc_nutation(s,fl=None):
+    logger.info(strm("pre-processing for p90 varied nutation"))
```

Comment:
FYI ,you only need `strm` if you're trying to print a variety of arguments, as you would in a print statement w/ commas